config/config.rush

```diff
@@ -1,3 +1,4 @@
 truncation-factor: false
 history-limit: false
 show-errors: true
+plugin: ./plugins
```

Comment:
That sounds like it would be helpful, users could get a "just works" experience without any config. Maybe `~/Library/Application Support/rush/plugins` on Mac and `~/.config/rush/plugins` on Linux.